.github/workflows/R-CMD-check.yaml

```diff
@@ -73,6 +73,12 @@ jobs:
           sudo apt install g++-6 -y
           echo $'CXX1X=g++-6\nCXX11=g++-6' >> ~/.R/Makevars
 
+      - name : Install compiler version
+        if: matrix.config.custom == 'gcc 4.8'
+        run: |
+          apt-get install -y g++-4.8
+          echo $'CXX1X=g++-4.8\nCXX11=g++-4.8' >> ~/.R/Makevars
```

Comment:
@jimhester `CXX1X` doesn't look right; did you mean to set the C compiler as well?

Reply:
No, it is correct `CXX1X` is what R used for C++11 on older R versions

Reply:
Not that we are using an older R version in this particular case, but mainly there for consistency's sake.